test/test.baseexternalclient.ts

```diff
@@ -484,6 +503,80 @@ describe('BaseExternalAccountClient', () => {
         scope.done();
       });
 
+      it('should resolve with the expected response on workforce configs with client auth', async () => {
```

Comment:
This isn't very descriptive.

something like:

'should use client auth over passing the workforce user project when both are provided'